detectable/src/main/java/com/synopsys/integration/detectable/detectables/cpan/CpanCliDetectable.java

```diff
@@ -15,7 +15,7 @@
 import com.synopsys.integration.detectable.extraction.ExtractionEnvironment;
 import com.synopsys.integration.executable.ExecutableRunnerException;
 
-@DetectableInfo(name = "Cpan CLI", language = "Perl", forge = "CPAN", accuracy = DetectableAccuracyType.HIGH, requirementsMarkdown = "File: Makefile.PL. Executable: cpan.")
+@DetectableInfo(name = "Cpan CLI", language = "Perl", forge = "CPAN", accuracy = DetectableAccuracyType.HIGH, requirementsMarkdown = "File: Makefile.PL. Executables: cpan, and cpanm.")
```

Comment:
Just to confirm, the CPAN detector requires both cpan **AND** cpanm? Or is it sufficient to have just one of them?

Reply:
> Both are required. `cpanm` helps us determine what modules the project needs, while `cpan` tells us what versions of modules are installed on the system.